Services/Accessibility/classes/class.ilAccessibilityControlConceptGUI.php

```diff
@@ -112,11 +114,14 @@ protected function showControlConcept()
             $document = $this->accessibilityEvaluation->document();
             $tpl->setVariable('ACCESSIBILITY_CONTROL_CONCEPT_CONTENT', $document->content());
         } else {
+            $mails = (ilAccessibilitySupportContacts::getMailsToAddress() != "")
+                ? ilUtil::prepareFormOutput(ilAccessibilitySupportContacts::getMailsToAddress())
+                : $this->settings->get("admin_email");
```

Comment:
Additionally, I added a fallback here if link for Accessibility Control Concept is enabled, but no Accessibility Point of Contact is defined, so the user does not run into a mailto without recipient.